README.md

```diff
@@ -58,11 +58,14 @@ at `window.videojs` and will throw an error if it does not exist.
   If you are using jQuery, this polyfill must be loaded and initialized before jQuery is
   initialized. Unfortunately, the Chromecast framework loads the `webcomponents.js`
   polyfill via a dynamically created `<script>` tag. This causes a race condition (see
-  #17). Setting `preloadWebComponents` to `true` will make this plugin add the
-  `webcomponents.js` polyfill synchronously when the polyfill is needed. If you use the
-  `preloadWebComponents: true` option, you should make sure that this plugin is loaded
-  before jQuery. Then include the Chromecast framework after this plugin as you normally
-  would.
+  #17). Also, including `webcomponents.js` anywhere on the page will break jQuery's fix for
+  bubbling some events to `document` (e.g. onchange events for `<select>`).
+  Setting `preloadWebComponents` to `true` will fix these 2 problems by making
```

Comment:
What do you think of changing this to something along the lines of the following?

> ... Also, the full version of `webcomponents.js` contains a polyfill for the shadow DOM which breaks jQuery's fix for bubbling some events to `document` (e.g. onchange events for `<select>`, see #21). Setting `preloadWebComponents` to `true` will fix these 2 problems by (1) making this plugin add the `webcomponents.js` polyfill synchronously when needed and (2) using the `webcomponents-lite.js` version as it does not include the shadow DOM polyfills. ...

Reply:
✔️